backend/alternative_flights/generate_flight_data.py

```diff
@@ -0,0 +1,95 @@
+import os
+import json
+import random
+from datetime import datetime, timedelta
+
+def generate_flight_data(start_date_str):
```

Comment:
You already had a script committed to generate data, right? Is this just for the test?

Reply:
I moved that file here because if I need to dockerize them later while deploying, I'll have all my python files together.